hello-world/simponic/.gitignore

```diff
@@ -0,0 +1,3 @@
+iso/*
```

Comment:
ignore everything in the `iso` folder? :smile_cat:

Reply:
Yeah it's all just GRUB stuff that's easily generated with the make file

Reply:
ahhh good catch :)

Reply:
I think you'll still want a `*.iso` target here too :)

Reply:
Wouldn't that ignore the precompiled demo?

Reply:
Yes, but this is desired! You can force-add changes with `git add -f`